platform/fabricx/core/finality/listenermanager.go

```diff
@@ -86,35 +100,44 @@ func (p *Provider) NewManager(network, channel string) (ListenerManager, error)
 		return nil, err
 	}
 
-	lm, err := newNotifi(p.baseCtx, cfg)
+	// Create a listen context that does not inherit the cancellation of the baseCtx.
+	// This is necessary because the baseCtx might be cancelled by a different component,
+	// but the listener should only stop if it's explicitly cancelled (e.g., during shutdown).
+	listenCtx := context.WithoutCancel(p.baseCtx)
```

Comment:
this is not clear to me. I thought we use the `baseCtx` to shutdown all managers if we shutdown the FSC node?